soccer/gameplay/tactics/one_touch_pass.py

```diff
@@ -20,59 +22,108 @@ class OneTouchPass(composite_behavior.CompositeBehavior):
     receivePointChangeThreshold = 0.15  # 15%
 
     class State(enum.Enum):
-        passing = 1
+        setup=1
+        passing = 2
 
-    def __init__(self):
+    def __init__(self,
+                 skillkicker=None):
         super().__init__(continuous=False)
 
+        if skillkicker == None:
+            skillkicker = skills.pivot_kick.PivotKick()
+
         self.tpass_iterations = 0
         self.force_reevauation = False
 
+        rp = self.calc_receive_point()
+        # self.add_subbehavior(skills.move.Move(main.ball().pos - (rp - main.ball().pos).normalized()*.4), 'passSetup', priority=5)
+        # self.add_subbehavior(skills.move.Move(rp), 'receiveSetup', priority=5)
```

Comment:
Why are we adding in commented-out code? Where did it originally come from? If it was an unmerged PR this should be deleted and the PR should get fixed, if it was at comp then it can probably be deleted anyway.

Reply:
It was comp where we did the move before the line kick, we refactored halfway through to make the move part of line kick so these are leftovers. They should just be removed